### ISSUE TITLE
Unable to recognize any Chinese characters even when chi_* language is selected

### ISSUE THREAD

**Issue by Lin-Buo-Ren** [jbarlow83/OCRmyPDF#208]
Only ASCII characters are recognized for some reason, I can recognize Chinese characters if uses the rescanned TIFF image directly with Tesseract.

## OCRmyPDF Command Line
```
ocrmypdf\
     --language chi_tra\
     --deskew\
     --clean\
     使用說明2.pdf\
     使用說明2.optimized.pdf
```

## Test File
https://drive.google.com/open?id=18XL4dczzO-xruQE2nQqdgxowg2w0HiPT

## Current Output
https://drive.google.com/open?id=1ujYkOG2B45l3jAszJZ9nRYktZaC7HA-Y

Console output: https://pastebin.com/KKEGb4pT

## Temporary Files
https://drive.google.com/open?id=1547AoTeZnn5b5rmoZ20JcpGDP6EuMAl6

## Rescanned TIFF (600DPI)
https://drive.google.com/open?id=1F9OzvQiteWFwaMZCVaJWjZ9-gA8Ms4a8

**Comment by jbarlow83** [comment]
Thanks for detailed listing of test files, that was very helpful.

I think the issue is that your version of Ghostscript (9.18) is stripping out the Chinese characters. It could be your version of Ghostscript doesn't have the required fonts or that a text-encoding bug fixed in 9.20 (which I reported) is responsible. If you use the ``--sidecar`` option the output file will contain all of the Chinese characters.
 
I have Ghostscript 9.22 and I can't reproduce the problem, so if you can upgrade Ghostscript that will probably fix it. Alternately, you could try `--output-type pdf` to output an ordinary PDF instead of PDF/A, which will skip using Ghostscript.

Aside: you'll probably get better results from Tesseract using ``--language chi_tra+eng`` to indicate that both English and Chinese are present. 

Let me know what you find and I'll add a warning about older versions of Ghostscript.

**Comment by Lin-Buo-Ren** [comment]
Thanks for the analysis!  I've upgrade the ghostscript and the issue is no longer reproduced.
